dev-libs/apache-arrow/apache-arrow-9.0.0-r2.ebuild

```diff
@@ -1,41 +1,46 @@
 # Copyright 2021-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
-PYTHON_COMPAT=( python3_{7..9} )
+PYTHON_COMPAT=( python3_{8..11} )
 
 inherit cmake python-single-r1
 
 IUSE="+bzip2 +lz4 +parquet +python +zlib +zstd"
-REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
 
-DESCRIPTION="A cross-language development platform for in-memory data."
+DESCRIPTION="A cross-language development platform for in-memory data"
 HOMEPAGE="https://arrow.apache.org/"
 SRC_URI="https://www.apache.org/dyn/closer.lua?action=download&filename=arrow/arrow-${PV}/${P}.tar.gz"
 
 LICENSE="Apache-2.0"
 SLOT="0"
-KEYWORDS="~amd64 ~x86"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
```

Comment:
Please note that these keywords must not be added due to dependencies `dev-cpp/xsimd` and `dev-libs/thrift` not keyworded on those platforms.

Reply:
Agreed.